site.js

```diff
@@ -178,7 +178,10 @@ $(document).ready(function (){
         {content: '0.18.x', start: '2016-10-16', end: '2017-10-16', py2:true},
         {content: '1.0.x -', start: '2016-12-26', end: '2021-12-23'}
     ],
-
+    'xarray':[
+        {content: '0.x Series', start: '2014-05-11', end: '2018-12-31', py2:true},
+        {content: '2018.1.0', start: '2019-01-01', end: '2022-12-31'}
```

Comment:
Is `2018.1.0` supposed to be a version number? We don't know exactly which version our first release without Python 2 support will be yet.

Reply:
You can add the logo without a timeline entry, if you like. I'm not a big fan of the timeline anyway - it's quickly getting too big, and it's already starting to get out of sync with reality (e.g. it looks like IPython 7 is out, which it isn't).

Reply:
Sounds good, let's add the logo without a timeline.

Reply:
timeline removed.